chromium_src/chrome/browser/ui/webui/tab_search/tab_search.mojom

```diff
@@ -36,4 +36,6 @@ interface PageHandler {
   UndoFocusTabs() => ();
   // Open a new tab for purchasing Leo premium.
   OpenLeoGoPremiumPage();
+  // Set the user pref to enable or disable tab focus with Leo.
+  EnableTabFocus(bool enable);
```

Comment:
Do we need some way of notifying the frontend that we've enabled/disabled the feature (i.e. if it happens in another window ect)

Reply:
TabSearchPageHandler::OnTabOrganizationFeaturePrefChanged calls page_->TabOrganizationEnabledChanged which would handle this for us using upstream code directly.